crates/notary/server/tee/README.md

```diff
@@ -1,3 +1,115 @@
-This folder contains the necessary files to build a Docker image for running the Notary Server on Intel SGX-enabled hardware.
+This folder contains the necessary files to build a Docker image for running the Notary Server on Intel SGX-enabled hardware. 
+
+## Compile the notary server for Intel SGX
+
+We are using [Gramine](https://github.com/gramineproject/gramine) to run the notary server on Intel SGX. Gramine allows to run the notary server in an isolated environment with minimal host requirements.
+
+The isolated environment is defined via the manifest template (`notary-server.manifest.template`).
+
+The notary server for SGX is compiled with an extra Rust feature flag `tee_quote`. This makes the notary server use an ephemeral private notary key for signing attestations (`private_key_pem_path: "/ephemeral/notary.key"`) and also adds the SGX *quote* to the notary server's `/info` end point.
+
+### CI 
 
 The container is built as part of the CI pipeline. For details on the build process, refer to the [CI workflow configuration](../../../../.github/workflows/ci.yml).
```

Comment:
`The container is built as part of the CI pipeline.` - this can be removed i think? (not sure which container it's referring to also)